### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by luzpaz** [comment]
@peternewman please review, TIA!

**Comment by luzpaz** [comment]
@vikivivi do you mind reviewing? TIA!

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -13019,6 +13069,8 @@ esitmations->estimations
 esitmator->estimator
 esitmators->estimators
 esle->else
+esnure->ensure
+esnures->ensures
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:13073]:
```text
esnured->ensured
```
https://github.com/search?q=esnured&type=code

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -16578,7 +16635,10 @@ highighter->highlighter
 highighters->highlighters
 highights->highlights
 highjack->hijack
+highligh->highlight
+highlighed->highlighted
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:16639]:
```text
highligher->highlighter
highlighers->highlighters
```
https://github.com/search?q=highligher&type=code  (31K hits)
https://github.com/search?q=highlighers&type=code

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -17888,6 +17951,7 @@ infrantryman->infantryman
 infrasctructure->infrastructure
 infrastrcuture->infrastructure
 infrastruture->infrastructure
+infrastucture->infrastructure
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:17954]:
```text
infrastuctures->infrastructures
```
https://github.com/search?q=infrastuctures&type=code

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -18635,6 +18699,7 @@ interogators->interrogators
 interopeable->interoperable
 interoprability->interoperability
 interperated->interpreted
+interpert->interpret
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:18702]:
```text
interpertation->interpretation
interpertations->interpretations
interperted->interpreted
interperters->interpreters
interperting->interpreting
interpertive->interpretive
interperts->interprets

misinterpert->misinterpret
misinterperted->misinterpreted
misinterperting->misinterpreting
misinterperts->misinterprets
```
https://github.com/search?q=interpertation&type=code
https://github.com/search?q=interpertations&type=code
https://github.com/search?q=interperted&type=code
https://github.com/search?q=interperters&type=code
https://github.com/search?q=interperting&type=code
https://github.com/search?q=interpertive&type=code
https://github.com/search?q=interperts&type=code

https://github.com/search?q=misinterpert&type=code
https://github.com/search?q=misinterperted&type=code
https://github.com/search?q=misinterperting&type=code
https://github.com/search?q=misinterperts&type=code

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -20695,6 +20768,8 @@ mintor->mentor, monitor, minor,
 mintored->mentored, monitored,
 mintoring->mentoring, monitoring,
 mintors->mentors, monitors,
+minue->minute, menu,
+minues->minutes, menus,
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:20772]: Additional suggestions
```text
minue->minute, minus, menu,
minues->minutes, minus, minuses, menus,
```

**Review by vikivivi — CHANGES_REQUESTED** [review]

**Comment by luzpaz** [comment]
@vikivivi added in [abef4f6](https://github.com/codespell-project/codespell/pull/2408/commits/abef4f6b944b8f10f86a03ef766d72632f6328e4). Please re-review

**Comment by vikivivi** [comment]
This one is missed. github collapsed the conversation when there are too many.
See above "1 hidden conversation. Load more..." 
```text
esnured->ensured
```
https://github.com/search?q=esnured&type=code

**Comment by luzpaz** [comment]
Thanks I'll pay better attention to that. I appreciate your thoroughness. Added forgotten requested revision in [ebfba42](https://github.com/codespell-project/codespell/pull/2408/commits/ebfba423c1fa1f3deb3da8d49a1df640c0b5ecd0).

**Pull request merged**